nvflare/fuel/utils/json_scanner.py

```diff
@@ -90,11 +92,15 @@ def __init__(self, json_data: dict, location=None):
             raise ValueError("json_data must be dict")
         self.location = location
         self.data = json_data
+        self.logger = logging.getLogger("JsonScanner")
 
     def _do_scan(self, node: Node):
         try:
             node.processor.process_element(node)
         except BaseException as e:
+            if not is_secure():
+                self.logger.error(traceback.format_exc())
```

Comment:
I think you don't need to do a logging here. Simply raise ConfigError here, then the Configer will handle the error logging already.

Reply:
We discussed the purpose of this. Keep this implementation.